packages/web-twig/src/Resources/components/Collapse/stories/CollapseHideTrigger.twig

```diff
@@ -10,14 +10,15 @@
         suspendisse consectetur finibus a accumsan, bibendum dolor sit amet congue rhoncus erat. Semper turpis, varius iaculis venenatis ut congue sollicitudin nulla,
         convallis nibh maximus erat id aliquam. Nec interdum, auctor elit vitae pulvinar fusce gravida curabitur odio leo, nunc a mauris auctor. Vel integer, placerat pulvinar
         nisi nunc donec felis erat quis, phasellus ac nec phasellus ac nibh. Scelerisque  fringilla, viverra nisl vel purus sed porttitor porttitor et iaculis commodo, vehicula urna purus ultrices.
-        <ButtonLink
+        <a
             href="javascript:void(0)"
             data-spirit-toggle="collapse"
             data-spirit-target="CollapseExampleIII"
             data-spirit-more
+            role="button"
         >
-            ...more
-        </ButtonLink>
+            …more
```

Comment:
🎉 

```suggestion
            … more
```

Reply:
rebased to your version of twig demo, so this was already done by you